Traceback (most recent call last):
  File "main.py", line 140, in <module>
  File "main.py", line 31, in get_driver
UnboundLocalError: local variable 'driver' referenced before assignment
Traceback (most recent call last):
  File "main.py", line 140, in <module>
  File "main.py", line 31, in get_driver
UnboundLocalError: local variable 'driver' referenced before assignment
Traceback (most recent call last):
  File "main.py", line 140, in <module>
  File "main.py", line 31, in get_driver
UnboundLocalError: local variable 'driver' referenced before assignment
